package.json

```diff
@@ -12,10 +12,10 @@
     "url": "https://github.com/babel/babel-eslint.git"
   },
   "dependencies": {
-    "babel-code-frame": "^6.22.0",
-    "babel-traverse": "^6.23.1",
-    "babel-types": "^6.23.0",
-    "babylon": "^6.17.0"
+    "babel-code-frame": "7.0.0-alpha.12",
```

Comment:
There's a deprecation for code-frame, https://github.com/babel/babel/pull/5808/files

https://github.com/babel/babel-eslint/blob/54ed517d2ecb3afb93ac338c019f8dff224918c2/index.js#L421